### PR TITLE
Official Page link added

### PR DESCRIPTION
**Pull request by rishabdesai**
Only official page links and details added. This can be elaborated.

### CODE REVIEW

File: Useful Content.md [hunk 1]
```diff
@@ -1,3 +1,20 @@
...
+| ------ | ------ |
+| Challenge Start Date | 28-June-2021 |
+| Challenge End Date | 1-October-2021 |
+
+#### Official Links
+
+* [Orientation Video](https://www.youtube.com/watch?v=G9LtP9HcNHM)
+* [Page for Latest events](https://sites.google.com/udacity.com/awsmachinelearningchallenge/home)
+* [FAQ](https://sites.google.com/udacity.com/awsmachinelearningchallenge/faqs)
+* Official Recommended Course:
```
Comment (xxprime) [inline Useful Content.md:17]: Maybe this is better? What do you think?

```suggestion
* Recommended Pre-requisite Courses:
```

File: Useful Content.md [hunk 2]
```diff
@@ -1,3 +1,20 @@
...
+    * [ Intro to Python Programming](https://www.udacity.com/course/introduction-to-python--ud1110)
```
Comment (xxprime) [inline Useful Content.md:18]:
```suggestion
    * [Intro to Python Programming](https://www.udacity.com/course/introduction-to-python--ud1110)
```

File: Useful Content.md [hunk 3]
```diff
@@ -1,3 +1,20 @@
 ##  Curated List of the Best Content to refer for AWS and related Stuff
 
 Open for Contributions! Please Fork and Star the Repository, and start sending in PRs adding the content here 🚀.
+
+#### Important Dates:
+
+| Details | Date |
+| ------ | ------ |
+| Challenge Start Date | 28-June-2021 |
+| Challenge End Date | 1-October-2021 |
```
Comment (xxprime) [inline Useful Content.md:10]:
```suggestion
| Challenge End Date | 11-October-2021 |
```

**Review by xxprime — CHANGES_REQUESTED** [review]
Hi @rishabdesai, this looks great! I have a couple of suggestions below, kindly check if it's applicable. Thanks!

**Review by xxprime — APPROVED** [review]
Great work @rishabdesai! 💯 

Ship it @Shivansh2407! 🚀

**Review by Shivansh2407 — APPROVED** [review]
Great work @rishabdesai :tada: LGTM :+1:

**Comment by Shivansh2407** [comment]
Thank you @rishabdesai for contributing! Keep Contributing :tada:

**Pull request merged**